src/main.ts

```diff
@@ -67,13 +69,18 @@ async function run() {
       if (!repository) {
         core.setFailed("❌ Expecting repository metadata.")
         return;
-      }
+      } else
       if (bodyContains || bodyDoesNotContain) {
         const PRBody = pull_request?.body;
         core.info("Checking body contents");
-        if (!PRBody) {
-          core.warning("⚠️ The PR body is empty, skipping checks");
-        } else {
+        if (!PRBody && allowEmpty) {
+            core.warning("⚠️ The PR body is empty, skipping checks");
+          
+        } else if(!PRBody && !allowEmpty){
+          core.setFailed(
+            "The PR body is empty. Please add info."
+          );
+        }else {
```

Comment:
Same as above. Flow will get here if it needs to, and unrelated to the issue, don't you think so?